test/Maestro/US/Manage_Family/adjustName_Kid.yml

```
@@ -0,0 +1,15 @@
+name: Adjust the name of the kid
+
+---
+- tapOn:
+    label: "Tap on pencil icon"
+    id: "penIcon"
+- tapOn: "First name"
+- inputText:
+    label: "Adjust kids firstname"
+    text: "ness"
+- tapOn: "Save"
+- assertVisible: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)